.github/workflows/run-ci-for-external-forks.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow cleans up any leftover projects created by e2e runs.
+
+name: Run CI on behalf of External Forks
+on:
+  workflow_dispatch:
+    inputs:
+      pr-number:
+        description: "The PR number to run CI on behalf of"
```

Comment:
~~The PR number of which behalf to run CI on~~
~~The PR number on behalf which of to run CI~~
~~The PR number half behon to which CI~~

Nah, you're good.

Reply:
🤣